docs/_content/standaard/catalogi/index.md

```diff
@@ -145,6 +145,17 @@ Bovendien gelden er beperkingen op verdere acties die uitgevoerd kunnen worden o
     * Er mag GEEN nieuw objecttype aangemaakt worden met een relatie naar een objecttype met `concept=false` (create)
     * Er mag GEEN nieuwe relatie worden gelegd tussen een objecttype en een objecttype met `concept=false` (update, partial_update)
 
+#### <a name="ztc-010">Relaties tussen objecten ([ztc-010](#ztc-010))</a>
```

Comment:
```suggestion
#### <a name="ztc-010">Relaties tussen objecttypen ([ztc-010](#ztc-010))</a>
```
Anders worden objecten misschien verward met `Zaak`, `Besluit`, etc.

Reply:
:+1: - was ook al een eerdere opmerking van @michielverhoef in een andere PR. Naar consistentie toe is dit beter.